tools/python_api/test/test_exception.py

```diff
@@ -1,4 +1,7 @@
 import pytest
+import sys
+sys.path.append('../build/')
```

Comment:
double check if this is needed.

Reply:
yes seems to be required, the tests failed after I removed it: https://github.com/kuzudb/kuzu/actions/runs/3838919604/jobs/6535990703